### PR TITLE
Update Bot description in /groups

### PR DESCRIPTION
**Pull request by MathyFurret**
Bots got nerfed 2 years ago but nobody updated /groups

### CODE REVIEW

**Comment by DaWoblefet** [comment]
While you're at it, you might as well change the position of bot in between voice and driver to be more consistent with sim ranks.

I don't know the policy on merging files with translations already written up. @AnnikaCodes?

File: server/chat-commands/info.ts [hunk 1]
```diff
@@ -1552,7 +1552,7 @@ export const commands: Chat.ChatCommands = {
 			`§ <strong>Section Leader</strong> - They oversee rooms in a particular section`,
 			`% <strong>Global Driver</strong> - Like Voice, and they can lock users and check for alts`,
 			`@ <strong>Global Moderator</strong> - The above, and they can globally ban users`,
-			`* <strong>Global Bot</strong> - Like Moderator, but makes it clear that this user is a bot`,
+			`* <strong>Global Bot</strong> - They can use HTML anywhere`,
```
Comment (AnnikaCodes) [inline server/chat-commands/info.ts:1555]: Should these continue to mention that the rank is intended for accounts controlled by software?

**Comment by AnnikaCodes** [comment]
> While you're at it, you might as well change the position of bot in between voice and driver to be more consistent with sim ranks.
> 
> I don't know the policy on merging files with translations already written up. @AnnikaCodes?

You should change the translation files to reflect the change to the main code. The translations can be left temporarily blank so that they appear in English (and will catch the notice of the translation team) or they can stay as the old translation since it seems to be close enough.

**Comment by AnnikaCodes** [comment]
@MathyFurret are you still working on this?

**Comment by MathyFurret** [comment]
been fixated on other things, will try to wrap this up within the next week or so

**Comment by AnnikaCodes** [comment]
Thanks!

**Pull request merged**